src/OrchardCore.Modules/OrchardCore.Contents/Views/ContentPartName.Edit.cshtml

```diff
@@ -0,0 +1,7 @@
+ <h5>
+    @Model.DisplayName
+    @if(!String.IsNullOrEmpty(Model.Description))
```

Comment:
Can we keep the ContentTypePartDefinition property as-is instead of having to project two properties? That will also removethe Arguments.From call.

Reply:
Edit templates in Liquid? I would not even know where to start to customize an editor in liquid. I think only supporting edit modes or custom parts is sufficient.

Reply:
May be just not introduce `ContentPartShape`  at all and let developer handle `DisplayName` by defining alternates for `ContentPart_Edit`?

Reply:
Done